app/views/cart/show.html.erb

```diff
@@ -32,6 +32,10 @@
         <center>
           <p><%=quantity%></p>
           <p><%= link_to "Remove", "/cart/#{item.id}",method: :delete %></p>
+          <br>
+          <p><%= button_to "+", "/cart/#{item.id}?add=true", method: :patch %></p>
+          <br>
+          <p><%= button_to "-", "/cart/#{item.id}?add=false", method: :patch %></p>
```

Comment:
Whoa whats that ```?add=true``` do? Is it a query param? I'd love to know more about this

Reply:
I think Arique and I found that somewhere when we were pairing on this view file... I'll try to find where I learned about it.

Reply:
From what I can see on how it is functioning, this adds an attribute into the params and while they appear to be booleans, I found out they were in fact strings that read true and false.  So once I understood that detail (thanks to a rubberducky session @AngelaGuardia held for me) I was able to get the if statement logic to function properly.